code/modules/martial_arts/combos/judo/wheelthrow.dm

```diff
@@ -0,0 +1,18 @@
+/datum/martial_combo/judo/wheelthrow
+	name = "Wheel Throw"
+	steps = list(MARTIAL_COMBO_STEP_GRAB, MARTIAL_COMBO_STEP_DISARM, MARTIAL_COMBO_STEP_HARM)
+	explaination_text = "From an armbar, flip your opponent over your shoulder, slamming them onto the floor, leaving them stunned."
+	combo_text_override = "Grab, Disarm, Harm"
+
+/datum/martial_combo/judo/wheelthrow/perform_combo(mob/living/carbon/human/user, mob/living/target, datum/martial_art/MA)
+	if(!IS_HORIZONTAL(target) || !target.IsImmobilized())
```

Comment:
This art doesn't check if the user is knocked down (unlike the other ones), is that intentional?

Reply:
This is intentional! As armbar does not apply new knockdown, this gives the attacker ~1-2 more seconds to clean up the combo! It used to be there, but I hit the combo all of once against an immobile testing skrell.